test/libyul/yulSyntaxTests/simple_functions.yul

```diff
@@ -0,0 +1,16 @@
+{
+    function a() {}
+    function f() { mstore(0, 1) }
+    function g() { sstore(0, 1) }
+    function h() { let x := msize() }
+    function i() { let z := mload(0) }
+}
+// ====
+// Dialect: ewasm
+// ----
+// DeclarationError: (41-47): Function not found.
```

Comment:
please either write a test that works or change the name of the file to reflect what we expect to fail